src/test/java/nl/tudelft/pixelperfect/event/EventFactoryTest.java

```diff
@@ -0,0 +1,71 @@
+package nl.tudelft.pixelperfect.event;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by woute on 5/10/2016.
+ */
+public class EventFactoryTest {
+
+  private EventFactory factory;
+
+  /**
+   * Setting up the EventFactory class for the test.
+   */
+  @Before
+  public void initialize() {
+    factory = new EventFactory();
+  }
+
+  /**
+   * Test if Id increases with each event.
+   */
+  @Test
+  public void testUniqueId() {
+    Event e1 = factory.create(0);
+    assertEquals(e1.getId(), 0);
+    Event e2 = factory.create(0);
+    assertEquals(e2.getId(), 1);
+  }
+
+  /**
+   * Test if an Astroid Field event is created.
+   */
+  @Test
+  public void testCreateAsteroidEvent() {
+    Event e1 = factory.create(0);
+    assertTrue(e1 instanceof AsteroidField);
```

Comment:
Not so sure whether instanceof is the best way to do this. You could for example use a method in the Event class (e.g. getType, returning a type identifier) that is implemented differently  by each subtype in order to verify them.